src/components/table/style-utilities.jsx

```diff
@@ -45,6 +45,9 @@ const borders = palette => {
     '&.border': {
       border: `1px solid ${palette.background.muted}`,
     },
+    '&.fatBorder': {
```

Comment:
Is the color-change intentional? Otherwise you can change only `border-width` instead